src/ResourceManager/Subscription/Commands.Subscription/Commands.Subscription.Netcore.csproj

```diff
@@ -39,7 +39,9 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication.Abstractions\Common.Authentication.Abstractions.Netcore.csproj" />
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Graph.RBAC\Common.Rbac.Netcore.csproj" />
     <ProjectReference Include="..\..\..\Common\Commands.Common\Common.Netcore.csproj" />
+    <ProjectReference Include="..\..\Common\Commands.Common.Authentication.ResourceManager\Common.ResourceManager.Authentication.Netcore.csproj" />
```

Comment:
Unfortunately, you cannot take a reference to this project, as it contains dependencies that are controlled by Profile, and may change.  I assume that this is to get the PSAzureSubscription model?

Reply:
If it is for this, we can move this type into another assembly (for example, the common resource manager assembly).

Reply:
Yes, it is just for PSAzureSubscription. I'll just create a new IAzureSubscription implementation instead for now and remove the dependency.